### ISSUE TITLE
Consumption new designer runs into renderComponentIntoRoot error

### ISSUE THREAD

**Issue by xuehongg** [Azure/LogicAppsUX#4471]
### Describe the Bug with repro steps 

Here is the error.

![image](<https://github.com/Azure/LogicAppsUX/assets/49498706/edd547d0-5095-4d34-8470-055f0a2cd6c6>)

If we click 'enable legacy designer', we can see the designer works well.

### What type of Logic App Is this happening in? 

Consumption (Portal)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

*No response*

### Browser 

chrome

### Additional context 

ICM 487126394

**Comment by rllyy97** [comment]
On the call with the customer we were seeing errors coming from our Designer code saying `Object.hasOwn` doesn't exist.  This should only occur with a lower node version that doesn't include this function, but if that were the case it should be erroring out for more than just one user like it is here.

**Comment by Eric-B-Wu** [comment]
From the Har logs: Chromium";v="92", " Not A;Brand";v="99", "Microsoft Edge";v="92". I think its just an outdated browser that the customer is using

**Comment by hartra344** [comment]
Yep, Edge got the API in version 93 in 2021. Every browser supported this API by 2022 so the customer is using an unsupported browser version in this case and should update.

https://caniuse.com/?search=Object.hasOwn
